### DISCUSSION TITLE
Make a Map Provider Default

### DISCUSSION THREAD

**Discussion by supermx1** [directus/directus#13170] [Feature Requests]
fHello, currently in Directus there is no way to make a map provider the default option. Like, let's say I add a google map provider and Mapbox provider to my Directus instance, from the project settings page. If I go to add new content, it defaults to Mapbox or OpenStreet map instead of Google Maps or whatever map provider is the default choice. Even after you select the map provider, if the browser is refreshed it falls back to Mapbox or OpenStreet map, you then have to go to the preferred map provider all over again.

**Comment by supermx1** [comment]
@sksar

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
